### ISSUE TITLE
Sending single element arrays into Bohrium causes errors with CPU VE

### ISSUE THREAD

**Issue by kenkendk** [bh107/bohrium#44]
Can be done with NumPy:

```
import numpy as np

m = np.empty(1, )
m[0] = np.float32(0.5)

print np.add.reduce(np.arange(10, dtype=np.float32) * m)
# Output is 0.0, expected 22.5

print np.add.reduce(np.arange(10, dtype=np.float32) * np.float32(0.5))
# Output is 22.5
```

**Comment by madsbk** [comment]
A simplified version of the same issue:

```python
import bohrium as np
m = np.ones(1, dtype=np.uint64)
a = np.arange(10, dtype=np.uint64) + m
np.flush()
print a
```
Which prints:
``[1 1 1 1 1 1 1 1 1 1]``

The correct result:
``[ 1  2  3  4  5  6  7  8  9 10]``

**Comment by madsbk** [comment]
Shape: [10], [10]
scalar: false
Sweeps:
Input views:
3:[ Dims: 1 Start: 0 Shape: 10 Stride: 0 Base=>[ Addr: 0x1831e40 Type: BH_UINT64 #elem: 1 Data: (nil) ]]
Output views:
4:[ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b5af50 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
Parameters:
[3][ Addr: 0x1831e40 Type: BH_UINT64 #elem: 1 Data: (nil) ]
[4][ Addr: 0x1b5af50 Type: BH_UINT64 #elem: 10 Data: (nil) ]
Constants:
[ CONST(BH_UINT64)=1 ]
[ CONST(BH_UINT64)=0 ]
Temp base-arrays:
[ Addr: 0x1b56bc0 Type: BH_UINT64 #elem: 10 Data: (nil) ]
[ Addr: 0x1b8df80 Type: BH_UINT64 #elem: 10 Data: (nil) ]
[ Addr: 0x1b956e0 Type: BH_UINT64 #elem: 10 Data: (nil) ]
Free base-arrays:
[ Addr: 0x1b7ffe0 Type: BH_UINT64 #elem: 10 Data: (nil) ]
Discard base-arrays:
[ Addr: 0x1b7ffe0 Type: BH_UINT64 #elem: 10 Data: (nil) ]
Sync base-arrays:
[1] BH_RANGE OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b956e0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[2] BH_FREE OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b7ffe0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[3] BH_DISCARD OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b7ffe0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[4] BH_MULTIPLY OPS=3{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b8df80 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
OP1 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b956e0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
OP2 [ CONST(BH_UINT64)=1 ]
}
[5] BH_FREE OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b956e0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[6] BH_DISCARD OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b956e0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[7] BH_ADD OPS=3{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b56bc0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
OP1 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b8df80 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
OP2 [ CONST(BH_UINT64)=0 ]
}
[8] BH_FREE OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b8df80 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[9] BH_DISCARD OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b8df80 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[10] BH_ADD OPS=3{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b5af50 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
OP1 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b56bc0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
OP2 [ Dims: 1 Start: 0 Shape: 10 Stride: 0 Base=>[ Addr: 0x1831e40 Type: BH_UINT64 #elem: 1 Data: (nil) ]]
}
[11] BH_FREE OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b56bc0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}
[12] BH_DISCARD OPS=1{
OP0 [ Dims: 1 Start: 0 Shape: 10 Stride: 1 Base=>[ Addr: 0x1b56bc0 Type: BH_UINT64 #elem: 10 Data: (nil) ]]
}

**Comment by madsbk** [comment]
Fixed in #140
